src/app/(auth)/sign-up/page.tsx

```
@@ -1,8 +1,17 @@
-import { Card } from "@/components/ui/card";
+import { auth } from "@/lib/auth";
 import { SignUpView } from "@/modules/auth/ui/views/sign-up-view";
+import { headers } from "next/headers";
+import { redirect } from "next/navigation";
+
+const page = async () => {
+    const session = await auth.api.getSession({
+        headers: await headers(),
+    })
+
+    if (!!session) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)